contracts/mvx-game-sc/README.md

```diff
@@ -1,3 +1,78 @@
-# lib
+# Game lobby SC
 
-`lib` is a simple Smart Contract.
+
+## Overview
+This smart contract covers the functionality of a basic game lobby under a set of rules imposed by the owner.
+
+**A user** can create a new wager game with the following specifications:
+- `waiting time`
+- `minimum number of players`
+- `maximum number of players`
+- `wager`
+
+and has to pay a fee for each new game. 
+
+**The owner** can:
+- `enable/disable` the contract for maintenance
+- set the `game starting fee` amount
+- set the `token id` for the currency of the SC (used for game starting fee, wager and reward)
+- `send rewards` or return the wager to the users who participated in a specific game
+
+Each player has to pay the `wager` amount set by the game creator in the `token id` set by the owner in order to join the game.
+
+The game is considered `invalid` until the `minimum number of players` have joined the game and if the `waiting time` has passed, no more players can join.
+
+The SC does not have any logic for calculating the winner, so it expects input from the owner with the winnes' addresses and the percentage (*100) of the total reward (sum of wagers) won by each. 
+
+**The game**:
+- If the game is `invalid`, the `wager` amount will be returned to the players that have joined the game and the `game starting fee` will be returned to the creator
+- If the game is `valid`, but no winners are sent by the owner, such in the case of a tie/draw, the contract will send back the `wager` amount paid by every player who joined
+- If the game is `valid` and winners are sent by the owner, the SC will send the reward to the winners as explained by the owner
```

Comment:
Maybe rephrase a little, as the owner does not explain anything.
Something like
If the game is `valid` and the winners were provided, the SC will send the rewards to them, based on the input of the owner.